auth0_flutter/android/src/main/kotlin/com/auth0/auth0_flutter/CredentialsManagerMethodCallHandler.kt

```diff
@@ -23,6 +24,8 @@ class CredentialsManagerMethodCallHandler(private val requestHandlers: List<Cred
     // GitHub issue: https://github.com/jacoco/jacoco/issues/1182
     @Generated
     lateinit var activity: Activity
```

Comment:
Do we still need this?

Reply:
Yes, as you can see `requireAuthentication` still needs `activity`